docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -1,83 +1,145 @@
 [[getting-started]]
-== Quick start: Add data using Elasticsearch APIs
+== Index and search data using {es} APIs
 ++++
-<titleabbrev>Basics: Add data using APIs</titleabbrev>
+<titleabbrev>Basics: Index and search using APIs</titleabbrev>
 ++++
 
-In this quick start guide, you'll learn how to do the following tasks:
+This quick start guide is a hands-on introduction to the fundamental concepts of Elasticsearch: <<documents-indices,indices, documents and field type mappings>>.
 
-* Add a small, non-timestamped dataset to {es} using Elasticsearch REST APIs.
-* Run basic searches.
+You'll learn how to create an index, add data as documents, work with dynamic and explicit mappings, and perform your first basic searches.
 
-[discrete]
-[[add-data]]
-=== Add data
-
-You add data to {es} as JSON objects called documents.
-{es} stores these
-documents in searchable indices.
+[TIP]
+====
+The code examples in this tutorial are in {kibana-ref}/console-kibana.html[Console] syntax by default.
+You can {kibana-ref}/console-kibana.html#import-export-console-requests[convert into other programming languages] in the Console UI.
+====
 
 [discrete]
-[[add-single-document]]
-==== Add a single document
+[[getting-started-prerequisites]]
+=== Prerequisites
 
-Submit the following indexing request to add a single document to the
-`books` index.
-The request automatically creates the index.
+Before you begin, you need to have a running {es} cluster.
+The fastest way to get started is with a <<run-elasticsearch-locally,local development environment>>.
+Refer to <<elasticsearch-intro-deploy,Run {es}>> for other deployment options.
 
 ////
 [source,console]
 ----
 PUT books
+PUT my-explicit-mappings-books
 ----
 // TESTSETUP
 
 [source,console]
 --------------------------------------------------
 DELETE books
+DELETE my-explicit-mappings-books
 --------------------------------------------------
 // TEARDOWN
 
 ////
 
+[discrete]
+[[getting-started-index-creation]]
+=== Step 1: Create an index
+
+Create a new index named `books`:
+
+[source,console]
+----
+PUT /books
+----
+// TEST[skip: index already setup]
+
+A `200` response indicates the index was created successfully.
```

Comment:
wonder if the console tester would blow up if you put the http code inline ... ex below